addons/l10n_it_edi_ndd/models/account_move.py

```diff
@@ -58,7 +58,12 @@ def _compute_l10n_it_payment_method(self):
     def _compute_l10n_it_document_type(self):
         document_type = self.env['l10n_it.document.type'].search([]).grouped('code')
         for move in self:
-            if move.country_code != 'IT' or move.l10n_it_document_type or move.state != 'posted':
+            if move.country_code != 'IT' or move.state != 'posted':
+                continue
+            # Setting a debit note with correct IT document type (initially set as the origin invoice's document type)
+            if move.debit_origin_id:
```

Comment:
`debit_origin_id` is defined in `account_debit_note`, which is not a dependency of this module :boom: